capstone/src/main/java/com/google/sps/servlets/calendar/AddEventsServlet.java

```diff
@@ -0,0 +1,85 @@
+package com.google.sps.servlets;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventDateTime;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet for adding events to a club's calendar */
+@WebServlet("/add-event")
+public class AddEventsServlet extends HttpServlet {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String startTime = request.getParameter(Constants.START_TIME_PROP) + Constants.TIMEZONE_OFFSET;
+    String endTime = request.getParameter(Constants.END_TIME_PROP) + Constants.TIMEZONE_OFFSET;
+    String title = request.getParameter(Constants.EVENT_TITLE_PROP);
+    String description = request.getParameter(Constants.EVENT_DESCRIPTION_PROP);
+    String clubName = request.getParameter(Constants.CLUB_PROP);
+    String calendarId = getCalendarId(clubName);
+
+    // Add event to calendar and return event information
+    try {
+      Event event = addEventToCalendar(calendarId, title, description, startTime, endTime);
+      response.setContentType("application/json");
+      response.getWriter().println(event.toString());
+    } catch (Exception error) {
+      System.out.println("Error: " + error);
+    }
+    response.sendRedirect(request.getHeader("referer"));
+  }
+
+  private String getCalendarId(String clubName) {
```

Comment:
I'm making a method in ServletUtil to get a property from a club (so you'd pass in a club's name and the property you're looking for). Just letting you know in case you want to use that for this after I submit and merge my PR (so it'd just be ServletUtil.getPropertyFromClub(clubName, Constants.CALENDAR_PROP)

Reply:
Awesome, thanks for letting me know!